kivy_ios/recipes/sdl3_mixer/__init__.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+import sh
+
+from kivy_ios.toolchain import Recipe, shprint
+
+
+class LibSDL3MixerRecipe(Recipe):
+    version = "78a2035cf4cf95066d7d9e6208e99507376409a7"
+    # url = "https://github.com/libsdl-org/SDL_ttf/releases/download/release-{version}/SDL3_ttf-{version}.tar.gz"
+    url = "https://github.com/libsdl-org/SDL_mixer/archive/{version}.tar.gz"
```

Comment:
That's literally the last commit from their main branch, I assume their 2.8.1 release from last month is missing some we need here

Reply:
`2.x.x` releases of sister libraries (`sdl_mixer`, `sdl_image`, `sdl_ttf`) are incompatible with SDL `3.x.x`  😃
The `2.x.x` releases are performed from a separate branch where only bug fixes and minor changes are applied.

Reply:
It's a pity they haven't tagged anything for v3 on the mixer yet, even if it's alpha